pkg/resourcemanager/controller/garbagecollector/reconciler.go

```diff
@@ -99,6 +101,10 @@ func (r *Reconciler) Reconcile(reconcileCtx context.Context, _ reconcile.Request
 		}
 	)
 
+	if pointer.BoolDeref(r.Config.ConsiderManagedResources, false) {
```

Comment:
Can you please enhance the GC integration test in `test/integration/resourcemanager/garbagecollector/garbagecollector_test.go`?

Reply:
Done